src/meta/test/CMakeLists.txt

```diff
@@ -17,35 +17,35 @@ nebula_add_test(
         gtest
 )
 
-nebula_add_test(
-    NAME
-        index_processor_test
-    SOURCES
-        IndexProcessorTest.cpp
-    OBJECTS
-        ${meta_test_deps}
-    LIBRARIES
-        ${ROCKSDB_LIBRARIES}
-        ${THRIFT_LIBRARIES}
-        ${PROXYGEN_LIBRARIES}
-        wangle
-        gtest
-)
+# nebula_add_test(
+#     NAME
```

Comment:
just one question ,why disable these tests?

Reply:
I  have open it